DB/common.php

```diff
@@ -2264,6 +2274,9 @@ function _convertNullArrayValuesToEmpty(&$array)
         }
     }
 
+    function lastInsertId() {
```

Comment:
I believe this is a duplicate of L2288 (a modification of the patch included in #276